src/VisualStudio/Core/Def/ColorSchemes/Enhanced.xml

```diff
@@ -0,0 +1,810 @@
+<Themes>
+    <!-- IMPORTANT: Any changes to this file need to be synced with the ColorSchemeApplier.ForegroundColorDefaulter. -->
```

Comment:
Moving to read values from this XML. Some values such as 'keyword' and 'operator' will still have to be hardcoded in the ColorDefaulter since they are not roslyn-owned classifications.